style.css

```
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  background-color: hsl(185, 75%, 39%);
+  align-items: center;
+  font-family: "Kumbh Sans", sans-serif;
+  background-image: url(/images/bg-pattern-top.svg),
+    url(/images/bg-pattern-bottom.svg);
+  background-position: right 50vw bottom 50vh, left 50vw top 50vh;
+  background-repeat: no-repeat;
+  
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: white;
+  width: 30%;
+  height: 60%;
+  border-radius: 1em;
+  overflow: hidden;
+}
+
+.pattern {
+  width: 100%;
+  height: 30%;
+  flex-basis: 40%;
+}
+
+.pattern img {
+  width: 100%;
+  height: 100%;
+}
+
+.user-image {
+  flex-basis: 40%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  position: relative;
+}
+
+.user-image img{
```

Ne smemo lepiti zagradu za klasu, treba ostaviti jedno prazno mesto

```suggestion
.user-image img {
```